osi_analytic_segments_stock_scrap/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2018 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Scrap Reason Code Analtyic",
+    "summary": "Provide a user-defined list of scrapping reasons",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Open Source Integrators",
+    "maintainer": "Open Source Integrators",
+    "website": "http://www.opensourceintegrators.com",
+    "category": "Inventory",
+    "depends": [
+        "stock_analytic",
+        "osi_scrap_reason_code",
```

Comment:
scrap reason code was moved to OCA which we should use. The module does work with using the OCA module instead so just changing this line is all that's needed.